ui/common/css/theme/gen/_mix.scss

```diff
@@ -0,0 +1,419 @@
+@mixin default-mix {
```

Comment:
this file is generated by ui/build and contains all the definitions for auto mixed colors. All these css variables are accessed via their scss wrappers in _wrap.scss.